curator-framework/src/main/java/org/apache/curator/framework/imps/EnsembleTracker.java

```diff
@@ -179,13 +179,16 @@ public static String configToConnectionString(QuorumVerifier data) throws Except
 
     private void processConfigData(byte[] data) throws Exception
     {
-        log.info("New config event received: " + Arrays.toString(data));
-
         Properties properties = new Properties();
         properties.load(new ByteArrayInputStream(data));
         QuorumMaj newConfig = new QuorumMaj(properties);
         currentConfig.set(newConfig);
 
+        if ( log.isInfoEnabled() )
```

Comment:
Instead of this check you can do:

```java
log.info("New config event received: {}", properties.toString());
```

or

```java
log.info(String.format("New config event received: %s", properties.toString()));
```